torch/csrc/jit/mobile/parse_bytecode.cpp

```diff
@@ -0,0 +1,135 @@
+#include <ATen/core/ivalue.h>
+#include <torch/csrc/jit/mobile/parse_bytecode.h>
+#include <torch/csrc/jit/mobile/type_parser.h>
+#include <torch/csrc/jit/serialization/import_export_constants.h>
+#include <torch/custom_class_detail.h>
+
+namespace torch {
+namespace jit {
+OpCode parseOpCode(const char* str);
+using c10::IValue;
+
+IValue expect_field(
+    IValue tup,
+    const std::string& expected_name,
+    size_t entry) {
+  auto row = tup.toTuple()->elements().at(entry).toTuple();
+  TORCH_INTERNAL_ASSERT(
+      row->elements().at(0).toStringRef() == expected_name,
+      "Expected ",
+      expected_name,
+      " found ",
+      row->elements().at(0).toStringRef());
+  return row->elements().at(1);
+}
+
+IValue Tup(std::vector<IValue> ivalues) {
```

Comment:
Should we rename these to something clearer like to_tuple or as_tuple (which is more consistent naming with the other functions in import_export_functions.h)?

Also, before these were static functions, probably unnecessarily?
How are we using them?
Should this code include import_export_functions.h?

Reply:
Make sense to rename it! 

Before it's only used in mobile/import.cpp. Now it's also used in the test driver to quickly build a bytecode on the fly. That's the reason they are moved here without static.

Good catch on the header include!